src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,7 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+
+@EnableJpaRepositories("com.baeldung.persistence.repo")
```

Comment:
These two annotations aren't needed, only when in need of scanning "non-default" src folders. Which is not the case.